defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
```

Comment:
I do agree having everything neatly in a list is an improvement. However it does confuse the way the role works since some of the things right now are declared via variables, which dynamically affect the args contents, but at the same time the whole list of args can just be declared at once. I do find these two ways of configuring do create some confusion on how the role should be used.

@robbyoconnor may I ask what is the use case of allowing to override the default arguments currently provided? As even now you can provide extra arguments, so it must be the need to change some of the current hard coded defaults? Could these be provided as separate configuration flags like some of the previous ones?

The reason I like the more easier to understand configuration flags is that the role then hides the complexity of certbot, allowing the user not to need to understand what flags need to be passed in. If we're missing some use case here that needs extra flags passed in to replace the current default, I would personally prefer overriding them via some configuration item.

Your thoughts on this @gronke ?